### PR TITLE
Update 04.baselines.md

### PR DESCRIPTION
**Pull request by khyu**
**Did you add yourself as a [contributor](https://github.com/Benjamin-Lee/deep-rules/blob/master/contributors.md) if this is your first contribution?**

- [x] Yes, I added myself or am already a contributor

**Any more details?**

### CODE REVIEW

File: content/04.baselines.md [hunk 1]
```diff
@@ -1 +1,10 @@
 ## Tip 2: Use traditional methods to establish performance baselines
+
+Before diving into a fancy thousand-layer neural network, always implement at least a simple model to establish an adequate performance baseline. 
+For example, researchers can build multinomial logistic regression or random forests models using the same software framework that is being used for DL and evaluate its classification performance.
```
Comment (Benjamin-Lee) [inline content/04.baselines.md]: "forests" -> "forest"

Comment (khyu) [reply content/04.baselines.md]: Thanks for the edit suggestion. I'll make the change accordingly.

File: content/04.baselines.md [hunk 2]
```diff
@@ -1 +1,10 @@
 ## Tip 2: Use traditional methods to establish performance baselines
+
+Before diving into a fancy thousand-layer neural network, always implement at least a simple model to establish an adequate performance baseline. 
+For example, researchers can build multinomial logistic regression or random forests models using the same software framework that is being used for DL and evaluate its classification performance. 
+This approach will help researchers with assessing the complexity of the task at hand and debugging more complex DL architectures. 
+Depending on the amount of available data and the type of tasks, DL models may not necessarily be the best performing one. 
+As an illustration, the Supplemental Materials of Rajkomar A et al. [@doi:10.1038/s41746-018-0029-1] indicated that their simple baseline models had achieved performance comparable with that of DL in a number of tasks, which may be a surprise to many. 
+It is worth noting that many conventional machine learning methods (such as support vector machines and random forests) require parameter tuning.
```
[the remark this answers is not in the document]
Comment (khyu) [reply content/04.baselines.md]: Thanks for the edit suggestion. I've made the change accordingly.

File: content/04.baselines.md [hunk 3]
```diff
@@ -1 +1,10 @@
 ## Tip 2: Use traditional methods to establish performance baselines
+
+Before diving into a fancy thousand-layer neural network, always implement at least a simple model to establish an adequate performance baseline. 
+For example, researchers can build multinomial logistic regression or random forests models using the same software framework that is being used for DL and evaluate its classification performance. 
+This approach will help researchers with assessing the complexity of the task at hand and debugging more complex DL architectures. 
+Depending on the amount of available data and the type of tasks, DL models may not necessarily be the best performing one. 
+As an illustration, the Supplemental Materials of Rajkomar A et al. [@doi:10.1038/s41746-018-0029-1] indicated that their simple baseline models had achieved performance comparable with that of DL in a number of tasks, which may be a surprise to many.
```
[the remark this answers is not in the document]
Comment (khyu) [reply content/04.baselines.md]: Thanks for the suggestion. I modified the sentence accordingly.

**Comment by agitter** [comment]
> Looks like there are conflicts with master?

It was the addition of the anchor to the header.  I resolved it in e5a56a2b264c66f6d3fe8df7d20e11661e189434

**Pull request merged**